astropy/io/votable/tree.py

```diff
@@ -1886,7 +1893,10 @@ def system(self, system):
             "barycentric",
             "geo_app",
         ):
-            warn_or_raise(E16, E16, system, self._config, self._pos)
+            if not self._config.get("version_1_5_or_later"):
+                # Starting in v1.5, system values come from the IVOA refframe vocabulary
+                # (http://www.ivoa.net/rdf/refframe).  For now we are not checking those values.
```

Comment:
I filed #16859 to add the validation of `system` against the external vocabulary.